io.catenax.secondary_material_content/1.0.0/SecondaryMaterialContent.ttl

```diff
@@ -0,0 +1,417 @@
+   samm:characteristic :PrimaryInorganic .
+
+:certificateName a samm:Property ;
+   samm:preferredName "certificate Name"@en ;
+   samm:description "If available, the name of a certificate verifying the recycling content can be included."@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "Redcert" .
+
+:certificateLink a samm:Property ;
+   samm:preferredName "certificateLink"@en ;
+   samm:description "The certificate can be referenced for example as a PDF file."@en ;
+   samm:characteristic samm-c:ResourcePath .
+
+:ReferencedStandard a samm-c:Enumeration ;
+   samm:preferredName "Referenced Standard"@en ;
+   samm:description "Norm authority who governs norm."@en ;
+   samm:dataType xsd:string ;
+   samm-c:values ( "AISI" "AS" "ASME" "DIN" "DIN EN" "EN" "JASO" "JIS" "GB" "ISO" "IS" ) .
+
+:UnitOfMeasureKeyTrait a samm-c:Trait ;
+   samm-c:baseCharacteristic :UnitReference ;
+   samm-c:constraint :AllowMassUnit .
+
+:UnitOfMeasureValue a samm-c:Quantifiable ;
+   samm:preferredName "Unit of Measure Value"@en ;
+   samm:description "Net mass of material in defined unit."@en ;
+   samm:dataType xsd:float .
+
+:BioBasedClass a samm-c:Enumeration ;
+   samm:preferredName "bio Based Class"@en ;
+   samm:description "Information on bio-based material class."@en ;
+   samm:dataType xsd:string ;
+   samm-c:values ( "gen1" "gen2" "gen3" ) .
+
+:SecondaryBioBased a samm:Characteristic ;
+   samm:preferredName "Secondary Bio Based"@en ;
+   samm:description "Secondary materials are recycled materials that can be used in manufacturing processes instead of or alongside virgin raw materials."@en ;
+   samm:dataType :SecondaryBioBasedEntity .
+
+:PrimaryBioBased a samm:Characteristic ;
+   samm:preferredName "Primary Bio Based"@en ;
+   samm:description "Primary materials are raw materials that are used for the first time during the planned process."@en ;
+   samm:dataType :PrimaryBioBasedEntity .
+
+:SecondaryInorganic a samm:Characteristic ;
+   samm:preferredName "Secondary Inorganic"@en ;
+   samm:description "Secondary materials are recycled materials that can be used in manufacturing processes instead of or alongside virgin raw materials."@en ;
+   samm:dataType :SecondaryInorganicEntity .
+
+:PrimaryInorganic a samm:Characteristic ;
+   samm:preferredName "Primary Inorganic"@en ;
+   samm:description "Primary materials are raw materials that are used for the first time during the planned process."@en ;
+   samm:dataType :PrimaryInorganicEntity .
+
+:UnitReference a samm:Characteristic ;
+   samm:preferredName "Unit Reference"@en ;
+   samm:description "Describes a Property containing a reference to one of the units in the Unit Catalog."@en ;
+   samm:see <https://openmanufacturingplatform.github.io/sds-documentation/bamm-specification/v1.0.0/appendix/unitcatalog.html#unit-catalog-units> ;
+   samm:dataType samm:curie .
```

Comment:
if you use curie as data type to reference a measurement unit, be aware of that it has the structure

unit:<measurement_unit> -> The regex, below will not work for that. So probably you might use xsd:string instead if you want to stick to the regex.